app/views/mortgage_calculator/land_transaction_taxes/_results_recalculation_form.html.erb

```diff
@@ -87,6 +87,7 @@
     </h2>
     <p class="stamp-duty__info-tip">
       <%= I18n.t("land_transaction_tax.next_steps.learn_more.tip_1") %>
+      <%= link_to  I18n.t("land_transaction_tax.next_steps.learn_more.tip_1_link_url") , I18n.t("land_transaction_tax.next_steps.learn_more.tip_1_link"), :target => "_blank" %>
```

Comment:
It's not critical, but our usual practice here has been to write this as something like: `<%= t("land_transaction_tax.next_steps.learn_more.tip_1_html", href: t('land_transaction_tax.next_steps.learn_more.tip_1_link')).html_safe %>` i.e. passing the URL from the view to the YAML. Then the translation file would need to include the markup, something like: `using the <a href="%{href}" target="_blank">paper return</a>.`